paychmgr/paych.go

```diff
@@ -149,57 +95,68 @@ func (pm *Manager) GetChannelInfo(addr address.Address) (*ChannelInfo, error) {
 
 // checks if the given voucher is valid (is or could become spendable at some point)
 func (pm *Manager) CheckVoucherValid(ctx context.Context, ch address.Address, sv *paych.SignedVoucher) error {
+	_, err := pm.checkVoucherValid(ctx, ch, sv)
```

Comment:
Moved the code in CheckVoucherValid() into an internal method so that we can return the paych.State (instead of loading the state repeatedly in different places)